worlds/pokemon_crystal/options.py

```diff
@@ -0,0 +1,374 @@
+from dataclasses import dataclass
+
+from Options import Toggle, Choice, DefaultOnToggle, Range, PerGameCommonOptions, NamedRange
+
+
+class Goal(Choice):
+    """Elite Four: collect 8 badges and enter the Hall of Fame
+        Red: collect 16 badges and defeat Red at Mt. Silver"""
+    display_name = "Goal"
+    default = 0
+    option_elite_four = 0
+    option_red = 1
+
+
+class JohtoOnly(Choice):
+    """Excludes all of Kanto, disables early Kanto access
+        Forces Goal to Elite Four unless Silver Cave is included.
```

Comment:
This is one of three lines you ended with a period, should probably remove them all
```suggestion
        Forces Goal to Elite Four unless Silver Cave is included
```